src/test/java/duke/tasks/DeadlineTest.java

```diff
@@ -0,0 +1,65 @@
+package duke.tasks;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.format.DateTimeParseException;
+
+import org.junit.jupiter.api.Test;
```

Comment:
Perhaps consider grouping line 3 and 7 together as a block?

Reply:
I can't do that because of the static declaration. Doesn't pass checkstyle :C